splitter/splitter.go

```diff
@@ -75,6 +75,22 @@ func (sc *SplitterConfig) XrpcRootUrl() string {
 	return "https://" + sc.UpstreamHost
 }
 
+func (sc *SplitterConfig) UpstreamUrl() string {
+	if strings.HasPrefix(sc.UpstreamHost, "http://") {
+		return "http://" + sc.UpstreamHost[7:]
+	}
+	if strings.HasPrefix(sc.UpstreamHost, "https://") {
+		return "https://" + sc.UpstreamHost[8:]
+	}
```

Comment:
It looks like the intention is to map to the websocket scheme:
```suggestion
	if strings.HasPrefix(sc.UpstreamHost, "http://") {
		return "ws://" + sc.UpstreamHost[7:]
	}
	if strings.HasPrefix(sc.UpstreamHost, "https://") {
		return "wss://" + sc.UpstreamHost[8:]
	}
```